src/Sulu/Bundle/MediaBundle/Media/FormatLoader/BaseXmlFormatLoader.php

```diff
@@ -0,0 +1,246 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\MediaBundle\Media\FormatLoader;
+
+use Sulu\Bundle\MediaBundle\Media\FormatLoader\Exception\InvalidMediaFormatException;
+use Symfony\Component\Config\Loader\FileLoader;
+use Symfony\Component\Config\Util\XmlUtils;
+
+/**
+ * Abstract xml loader for the image formats which contains the common part of all versions.
+ */
+abstract class BaseXmlFormatLoader extends FileLoader
+{
+    const XML_NAMESPACE_URI = 'http://schemas.sulu.io/media/formats';
+
+    const SCHEMA_URI = '';
+
+    const SCALE_MODE_DEFAULT = 'outbound';
+
+    /**
+     * @var \DOMXPath
+     */
+    protected $xpath;
+
+    /**
+     * @var array
+     */
+    private $defaultOptions = [];
+
+    /**
+     * @return array
+     */
+    public function getDefaultOptions()
+    {
+        return $this->defaultOptions;
+    }
+
+    /**
+     * @param array $defaultOptions
+     */
+    public function setDefaultOptions($defaultOptions)
+    {
+        $this->defaultOptions = $defaultOptions;
+    }
```

Comment:
What are these methods for? Should they really be called from the outside? Wouldn't it be better to make the `protected`?